tests/ExampleApps/HeadlessLibAndApp/Application/main.cpp

```diff
@@ -42,8 +42,8 @@ int main( int argc, const char* argv[] ) {
         viewer.setDataFileName( filename );
     }
     viewer.loadScene();
-    viewer.compileScene();
     viewer.setCamera();
+    viewer.compileScene();
```

Comment:
Is the order here important ? if so maybe add a comment
```suggestion
// need to be called after setCamera() because ...TODO
    viewer.compileScene();
```

Reply:
You're right. done in the last push.